client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,8 +1,19 @@
+import { useEffect } from 'react';
 import Grid from '@material-ui/core/Grid';
+import { useSocket } from '../../context/useSocketContext';
 import useStyles from './useStyles';
 
 export default function Dashboard(): JSX.Element {
   const classes = useStyles();
+  const { initSocket, disconnect } = useSocket();
+
+  useEffect(() => {
```

Comment:
Why don't we move this `useEffect` into the `useSocketContext` - even if we navigate from the `Dashboard` to any other view we will still be connected to the socket :)

Reply:
Made the suggested change.